web/src/scripts/webflow/licenseSync.mjs

```diff
@@ -205,8 +205,40 @@ const createNewLicenses = async () => {
   console.info(`Created a total of ${newLicenses.length} licenses`);
 };
 
+const getUnUsedLicenses = async () => {
+  const localLicensesRaw = loadAllLicenses();
+  const localLicensesFiltered = localLicensesRaw.filter((it) => it.webflowId === undefined);
```

Comment:
[sand] We can see it's filtered with .filter(). It's more important to denote what you're filtering _for_.